BfA/TheMotherlode/Trash.lua

```diff
@@ -149,29 +202,17 @@ function mod:EarthShieldApplied(args)
 	end
 end
 
--- Stonefury
-do
-	local engagedStonefury = {}
-	function mod:StonefuryInteract(args)
-		if self:MobId(args.sourceGUID) == 130635 then
-			engagedStonefury[args.sourceGUID] = true
-		elseif self:MobId(args.destGUID) == 130635 then
-			engagedStonefury[args.destGUID] = true
-		end
-	end
-
-	function mod:StonefuryAuraApplied(args)
-		local destGUID = args.destGUID
-		if self:MobId(destGUID) == 130635 and args.sourceGUID ~= destGUID then
-			engagedStonefury[destGUID] = true
-		end
-	end
+function mod:EarthShield(args)
+	self:Message2(args.spellId, "red", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "warning", "interrupt")
+end
 
-	function mod:FuriousQuake(args)
-		if engagedStonefury[args.sourceGUID] then
-			self:Message2(args.spellId, "orange", CL.casting:format(args.spellName))
-			self:PlaySound(args.spellId, "warning", "interrupt")
-		end
+-- Stonefury
+function mod:FuriousQuake(args)
+	local unit = self:GetUnitIdByGUID(args.sourceGUID)
```

Comment:
The messages out of combat should have already been fixed with #401. The advantage to the way it's handled there is that the mob does not need to have a unit id (no nameplate/nobody is targeting it) for it to be registered as in combat.

Is the issue still occurring? If it is, switching to this method is fine since it's probably not worth putting too much effort into.

Edit: On second thought, it's probably worth it just for simplicity.